com.one_ware.OneWare.yml

```diff
@@ -0,0 +1,49 @@
+app-id: com.one_ware.OneWare
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: oneware
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --persist=.local/share/OneWareStudio
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --filesystem=host
+modules:
+  - name: oneware
+    buildsystem: simple
+    build-commands:
+      - install -D apply_extra ${FLATPAK_DEST}/bin/apply_extra
+      - install -D -m755 oneware ${FLATPAK_DEST}/bin/oneware
+      - install -D -m644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -D -m644 ${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -D -m644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: extra-data
```

Comment:
why do you use extra-data? You said you were the upstream developer. So that imply you allow redistribution (non withstanding the " It is a free software "), so extra-data should not be used here.

Reply:
While the IDE System I made is open source, the Studio itself (including FPGA hardware features) is not.
Thats why the sources are downloaded from our cdn.

Reply:
note that I used the word "redistribution" here. A Free Software license would imply it, but it's not required. Since you are the upstream developer you should know whether it can be reditributed.

The use of `extra-data` is only for situation where things can be redistributed.

Reply:
I see, guess I missed that in the documentation. I was following an example from another flatpak that is also using Avalonia. So I guess I should move away from extra-data and instead download the sources in the build process. I will try this again later